packages/headless/src/controllers/breadcrumb-manager/headless-breadcrumb-manager.ts

```diff
@@ -294,18 +353,18 @@ export function buildBreadcrumbManager(
         categoryFacetBreadcrumbs: getCategoryFacetBreadcrumbs(),
         numericFacetBreadcrumbs: getNumericFacetBreadcrumbs(),
         dateFacetBreadcrumbs: getDateFacetBreadcrumbs(),
+        staticFilterBreadcrumbs: getStaticFilterBreadcrumbs(),
         hasBreadcrumbs: hasBreadcrumbs(),
       };
     },
 
     deselectAll: () => {
       dispatch(deselectAllFacets());
+      dispatch(deselectAllStaticFilters());
```

Comment:
I started by creating a dedicated action, to keep the concept of Facets and StaticFilters separate.

However, this creates a maintenance problem for the project because `deselectAllFacets` is also dispatched before a search. So, it means that we should also dispatch this new function there. Developers working at the action level would also need to dispatch two actions instead of one.

Instead, we could:
- Piggy-back on `deselectAllFacets`. My only issue with this is it makes Static Filters seem like facets, which they are not.
- Create a new action like `deselectUserFilters` or `deselectManualFilters` and dispatch that. We could deprecate `deselectAllFacets` or leave it as it is.

Reply:
I'm worried that in some cases, static filters may be used for somethingg that isn't meant to be deselected before a new search.

Perhaps another alternative could be to add a "autoDeselectOnNewSearch" option to facets and static filters, and make an action that deselects all of those exclusively.

Reply:
To be consistent perhaps the search box should dispatch the  `deselectAllStaticFilters` action too